app/policies/image_policy.rb

```diff
@@ -1,38 +1,26 @@
 class ImagePolicy < ApplicationPolicy
   class Scope < Scope
     def resolve
-      # return scope.all if user.present?
+      return scope.all.order(:title) if user&.role? :admin
       scope.where(disabled: false).order(:title)
     end
   end
 
   def create?
-    return false unless user.present?
-    true
+    user&.role? :admin
   end
 
-  def show?
-    return scope.where(id: record.id).exists? if user.present?
-
-    scope.where(id: record.id, disabled: false).exists?
-  end
-
-  def update?
-    return false unless user.present?
-    user.owner_of? record
+  def destroy?
+    user&.role? :admin
   end
 
-  def edit?
-    return false unless user.present?
-    user.owner_of? record
-  end
+  def show?
+    return !record.disabled? unless user.present?
 
-  def destroy?
-    return false unless user.present?
-    user.owner_of? record
+    user.role? :admin
   end
 
   def permitted_attributes
-    [:title, :tags, :source, :disabled]
+    [ :title, :tags, :source, :disabled ]
```

Comment:
hm, what does this do? Allow role based permitted attributes? I might have to build that...

Reply:
https://github.com/varvet/pundit#strong-parameters